frontend/src/app/modules/imports/components/import_report/import_report.component.html

```diff
@@ -80,14 +80,10 @@ <h3>
               {{ importData?.format_source_file }}
             </p>
             <p>
-              <b>Nombre de lignes :</b>
-              {{ importData?.source_count || 0 }}
+              <b>Nombre de lignes importées :</b>
```

Comment:
pari
```suggestion
              <b>Nombre de lignes partiellement ou totalement importées :</b>
```
Je sais pas trop si ça clarifie ou si ça complique de préciser ça mais je me dis que nombre de lignes importées ça peut être trompeur …

Reply:
Pas trop fan... Le problème c'est que la stat est pertinente surtout pour des imports avec une ligne == un entité..